modules/repository/repo.go

```diff
@@ -127,11 +127,23 @@ func MigrateRepositoryGitData(ctx context.Context, u *models.User, repo *models.
 	}
 
 	if opts.Mirror {
+		var interval = setting.Mirror.DefaultInterval
+		if opts.MirrorInterval != "" {
+			interval, err = time.ParseDuration(opts.MirrorInterval)
+			if err != nil {
+				log.Error("Failed to set Interval: %v", err)
+				return repo, err
+			}
+			if interval != 0 && interval < setting.Mirror.MinInterval {
+				log.Error("Interval is too frequent. %v", err)
+				return repo, err
```

Comment:
err will always be nil here - you need to create an err to return

Reply:
Done,